staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -301,7 +319,12 @@ func NewInformer(
 			// from oldest to newest
 			for _, d := range obj.(Deltas) {
 				switch d.Type {
-				case Sync, Added, Updated:
+				case Sync:
```

Comment:
This will obviously will break a lot of clients that rely on the current behavior.
This won't be easy to add it now...

@deads2k @lavalamp

Reply:
@wojtek-t  that was my concern too...but I believe this will be more beneficial in the long run to separate out the sync and update than to keep in tightly  coupled.